src/NServiceBus.RabbitMQ/RabbitMQTransportInfrastructure.cs

```diff
@@ -62,7 +86,7 @@ public override TransportSendInfrastructure ConfigureSendInfrastructure()
         {
             return new TransportSendInfrastructure(
                 () => new MessageDispatcher(channelProvider),
-                () => Task.FromResult(StartupCheckResult.Success));
+                () => Task.FromResult(DelayInfrastructure.CheckForInvalidSetting(routingTopology is ISupportNativeDelays, disableTimeoutManager)));
```

Comment:
I can see it either way. We only need it in those two places at startup, so is adding an extra bool to the class to keep the value around worth it?